### ISSUE TITLE
Adding Header to Notification Page

### ISSUE THREAD

**Issue by omergulen** [bluesky-social/social-app#971]
**Is your feature request related to a problem? Please describe.**

The content is too close to the top of the screen and it is not intuitive to have a page where there is no title showing where we are.

I'd like to add a Header to Notifications page, simple and same as Moderation page Header.

Current Notifications Page Header (none):
<img width="540" alt="Screenshot 2023-07-04 at 20 43 12" src="https://github.com/bluesky-social/social-app/assets/26525137/ce3e2ef7-3ac1-44f5-9948-88a5d994a8fc">

Current Moderation Page Header:
<img width="619" alt="Screenshot 2023-07-04 at 20 43 16" src="https://github.com/bluesky-social/social-app/assets/26525137/14957477-d57b-4150-9599-9a4445aefa2d">

**Comment by pfrazee** [comment]
I've got a PR not pushed up yet that adds tabs to the notifications, and that tab header should behave more closely to this expectation. Let me see if I can do that instead.

(As I think you saw, there are weird challenges around desktop layout sizing)

**Comment by omergulen** [comment]
Yeah, that'd be great, and yes unfortunately there were some weird challenges 🥲

**Comment by surfdude29** [comment]
As mary said, I think this is now fixed
